tests/device_tests/test_protect_call.py

```diff
@@ -18,92 +18,53 @@
 
 import pytest
 
-from trezorlib import messages as proto
+from trezorlib import btc, messages as proto
 from trezorlib.exceptions import PinException
 
 # FIXME TODO Add passphrase tests
 
 
 @pytest.mark.skip_t2
 class TestProtectCall:
-    def _some_protected_call(self, client, button, pin, passphrase):
+    def _some_protected_call(self, client):
         # This method perform any call which have protection in the device
-        res = client.ping(
-            "random data",
-            button_protection=button,
-            pin_protection=pin,
-            passphrase_protection=passphrase,
-        )
-        assert res == "random data"
+        res = btc.get_address(client, "Testnet", [0])
+        assert res == "mndoQDWatQhfeQbprzZxD43mZ75Z94D6vz"
 
     @pytest.mark.setup_client(pin="1234", passphrase=True)
     def test_expected_responses(self, client):
-        # This is low-level test of set_expected_responses()
-        # feature of debugging client
-
-        with pytest.raises(AssertionError), client:
-            # Scenario 1 - Received unexpected message
-            client.set_expected_responses([])
-            self._some_protected_call(client, True, True, True)
-
-        with pytest.raises(AssertionError), client:
-            # Scenario 2 - Received other than expected message
-            client.set_expected_responses([proto.Success()])
-            self._some_protected_call(client, True, True, True)
-
-        with pytest.raises(AssertionError), client:
-            # Scenario 3 - Not received expected message
-            client.set_expected_responses(
-                [proto.ButtonRequest(), proto.Success(), proto.Success()]
-            )  # This is expected, but not received
-            self._some_protected_call(client, True, False, False)
-
-        with pytest.raises(AssertionError), client:
+        with client:
```

Comment:
if the `expected_responses` scenarios are removed, is there even a point in keeping this testcase?

Reply:
Hmm probably not. Removed